auto3dseg/algorithm_templates/swinunetr/scripts/algo.py

```diff
@@ -280,7 +297,6 @@ def objective(trial):
                 "validation_data_device", ["cpu", "gpu"]
             )
             device_factor = 2.0 if validation_data_device == "gpu" else 1.0
-            ps_environ = os.environ.copy()  # ensure the CUDA_VISIBLE_DEVICES is copied when used.
```

Comment:
Instead of deleting it, maybe we should pass it to `subprocess.run(..., environ=ps_environ)`?

Reply:
thanks, added it back

Reply:
@heyufan1995 , sorry I mistakenly told you the argument for the `subprocess.run`. It should be `env` instead of `environ`:
```
subprocess.run(cmd_list, env=ps_environ, check=True)
```